app/models/user.rb

```diff
@@ -67,18 +67,9 @@ class User < ApplicationRecord
   end
```

Comment:
I think you should also hide all the columns that have been moved over, this way if any of the tests exercise some code that are still trying to use those columns, they will break.

See https://api.rubyonrails.org/classes/ActiveRecord/ModelSchema/ClassMethods.html#method-i-ignored_columns-3D

You can find an example here

https://github.com/forem/forem/blob/ff3be35d6e153b113493b48f75a96297d7cdaa26/app/models/podcast_episode.rb#L1-L4

Reply:
perfect point thank you